tests/dummy/app/initializers/router-ext.js

```diff
@@ -10,12 +12,28 @@ function initialize() {
   Ember.Router.reopen({
```

Comment:
FWIW, this reopen should be done outside of the `initialize` function (and we should remove the `hasRan` guard since the module will only ever be evaluated once).

Reply:
@rwjblue it already there, I just applied changes to update ember-cli, it's just a dummy app. It should be part of modernizing the ember engines project (that is "forgotten" for a long time), can we follow with this solution and make this upgrade simple, please?

Reply:
This still needs to get fixed. It's totally fine to me if that happens in a separate PR though.

Maybe make an issue tracking the fix and link it here? That way we won't forget...